autobuild/templates/dockercompose.j2

```diff
@@ -8,18 +8,32 @@ services:
     expose:
       - {{ daemon.rpcPort }}
       - {{ daemon.p2pPort }}
+{% if deploy_nevm and daemon.name == 'SYS' %}
+      - 8545
+      - 30303
+    ports:
+      - "8369:8369"
+      - "8545:8545"
+      - "30303:30303"
```

Comment:
@walkjivefly 
```yaml
    ports:
       - "8369:8369"
        - "8545:8545"
        - "30303:30303"
```
SidGrip just pinged me about ports 8545 and 30303 of the SYS container being opened to the outside world in the latest `docker-compose.yml`. He is opening those same two ports to the outside world for his GETH because they are the default ports for GETH access and he wants to give us access to GETH for testing.
I actually don't see why we need to open those two ports to the outside on the SYS container. Afaiu, XQ and Hydra both access SYS NEVM on the private *backend* network of docker, so they shouldn't require those ports to be open to the outside linux env. We should test this, of course.